target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -287,6 +287,7 @@ ramips_setup_interfaces()
 			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "7t@eth0"
 		;;
 	mlwg2|\
+	tl-wa801nd-v5|\
```

Comment:
Please move the board to the block around line 40, which disables the switch and sets eth0 as lan interface.

Reply:
But this will disable the switch function and vlan tagging using switch function ?

Reply:
Yes it disables the switch. A switch for a single port device doesn't really makes sense, does it?

Even with the current configuration you are sending untagged packets. We are using the switch vlan functionality normally to address individual ports of the switch, which isn't necessary in your case.

Reply:
Yes of course, but I did think the vlan functionnality was done in hardware mode if "switch" is enabled.

Code modified.